### PR TITLE
[SOAR-8703] Fix for dl get

### PR DESCRIPTION
**Pull request by cmcnally-r7**
### Description

Describe the proposed changes:

  - Changed dlGet to only retrieve 'data' from result

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

**Review by blaxminarayan-r7 — APPROVED** [review]

File: plugins/cisco_umbrella_destinations/plugin.spec.yaml [hunk 1]
```diff
@@ -351,11 +351,11 @@ actions:
       success:
         title: Success
         description: Successful returned value
-        type: destinationsEntity
+        type: '[]destinationsEntity'
         required: true
   dAdd:
-    title: Add Destinations to Destination List
-    description: Add a list of destinations to a destination list
+    title: Add a destination to a Destination List
```
Comment (mrinehart-r7) [inline plugins/cisco_umbrella_destinations/plugin.spec.yaml:357]:
```suggestion
    title: Add Destination to Destination List
```

**Review by mrinehart-r7 — CHANGES_REQUESTED** [review]
Two things to address here:
1. Make sure to bump the version in the plugin.spec.yaml and regenerate :)
2. We have a function that does the same thing as `return_non_empty` in the plugin runtime code - `clean`. You can use it like `from insightconnect_plugin_runtime.helper import clean`. I would suggest using that.

File: plugins/cisco_umbrella_destinations/bin/icon_cisco_umbrella_destinations [hunk 2]
```diff
@@ -6,7 +6,7 @@ from sys import argv
 
 Name = "Cisco Umbrella Destinations"
 Vendor = "rapid7"
-Version = "1.0.0"
+Version = "1.0.1"
```
Comment (blaxminarayan-r7) [inline plugins/cisco_umbrella_destinations/bin/icon_cisco_umbrella_destinations:9]: This should be 1.0.0, It is not released yet

File: plugins/cisco_umbrella_destinations/help.md [hunk 3]
```diff
@@ -42,9 +42,9 @@ Example input:
 
 ### Actions
 
-#### Add Destinations to Destination List
+#### Add Destination to Destination List
 
-This action is used to add a list of destinations to a destination list.
+This action is used to add a destination to a destination list.
```
Comment (blaxminarayan-r7) [inline plugins/cisco_umbrella_destinations/help.md:47]:
```suggestion
This action is used to add a destination to the destination list.
```

File: plugins/cisco_umbrella_destinations/help.md [hunk 4]
```diff
@@ -474,6 +474,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.0.1 - Updated action title (dAdd) and included an icon runtime helper function
```
Comment (blaxminarayan-r7) [inline plugins/cisco_umbrella_destinations/help.md:477]: You dont need this as, 1.0.0 is not released yet.

File: plugins/cisco_umbrella_destinations/icon_cisco_umbrella_destinations/actions/dAdd/schema.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 
 class Component:
-    DESCRIPTION = "Add a list of destinations to a destination list"
+    DESCRIPTION = "Add a destination to a destination list"
```
Comment (blaxminarayan-r7) [inline plugins/cisco_umbrella_destinations/icon_cisco_umbrella_destinations/actions/dAdd/schema.py:7]:
```suggestion
    DESCRIPTION = "Add a destination to the destination list"
```

File: plugins/cisco_umbrella_destinations/plugin.spec.yaml [hunk 6]
```diff
@@ -3,7 +3,7 @@ extension: plugin
 products: [insightconnect]
 name: cisco_umbrella_destinations
 title: Cisco Umbrella Destinations
-version: 1.0.0
+version: 1.0.1
```
Comment (blaxminarayan-r7) [inline plugins/cisco_umbrella_destinations/plugin.spec.yaml:6]: No version bump is needed.

File: plugins/cisco_umbrella_destinations/setup.py [hunk 7]
```diff
@@ -3,7 +3,7 @@
 
 
 setup(name="cisco_umbrella_destinations-rapid7-plugin",
-      version="1.0.0",
+      version="1.0.1",
```
Comment (blaxminarayan-r7) [inline plugins/cisco_umbrella_destinations/setup.py:6]: No version bump is needed

**Review by blaxminarayan-r7 — CHANGES_REQUESTED** [review]

**Pull request merged**